pkg/sql/schema_changer.go

```diff
@@ -1482,6 +1464,11 @@ type SchemaChangerTestingKnobs struct {
 	// transaction is unable to commit because it is violating the two
 	// version lease invariant.
 	TwoVersionLeaseViolation func()
+
+	// SkipComputedColumnSwapCleanup determines if the cleanup job should be ran for
+	// the computed column swap. If the cleanup job is not run, the old columns
+	// and indexes will not be cleaned up.
+	SkipComputedColumnSwapCleanup bool
```

Comment:
This testing knob feels pretty awkward -- is it possible to do something with the backfill notification like in the other tests for the test you are using this in?

Reply:
I tried the backfill knobs and but I don't think it supports this case - need to let the schema change "finish" but not let child jobs run. I added a new RunBeforeChildJobs knob which I think aligns more with the existing knobs, let me know if you think this makes sense.